src/app/admin/_utils/transformSpecialistFormData.js

```diff
@@ -0,0 +1,42 @@
+import { mapIdArrayToIdObjects } from '@/app/admin/_utils/mapIdArrayToIdObjects';
+
+function transformPlacesOfWork(placesArray) {
```

Comment:
🟠 these are veeeery specific ( it can be used in one place and requires very specific arguments for work ) and used in one place, is it really worth putting separately in `utils` ?